kolena/_utils/state.py

```diff
@@ -24,9 +24,8 @@
 
 import requests
 
-import kolena
```

Comment:
What's the reason for this change? There are a couple usages in this file with `kolena.xxx`. Let's revert the change if it is not needed.

Reply:
The pre-commit hooks removed this line. Now that I look at it, I'm not entirely sure why the `kolena.xxx` usages still work with `import kolena` removed.